resources/assets/js/components/FindSkillsModal.tsx

```diff
@@ -64,18 +69,32 @@ const messages = defineMessages({
     id: "findSkillsModal.searchResultsTitle",
     defaultMessage: `There are {numOfSkills} results for skills related to "{searchQuery}".`,
   },
+  searchBarInputLabel: {
+    id: "findSkillsModal.seachBarInputLabel",
+    defaultMessage: "Search for skills by name:",
+  },
+  searchBarInputPlaceholder: {
+    id: "findSkillsModal.searchBarInputPlaceholder",
+    defaultMessage: "eg. User interface design.",
+  },
+  searchBarButtonLabel: {
+    id: "findSkillsModal.searchBarButtonLabel",
+    defaultMessage: "Search Skills",
+  },
 });
 
 interface FindSkillsModalProps {
   oldSkills: Skill[];
   portal: "applicant" | "manager";
+  skills: Skill[];
   skillCategories: SkillCategory[];
   handleSubmit: (values: Skill[]) => Promise<void>;
 }
 
 const FindSkillsModal: React.FunctionComponent<FindSkillsModalProps> = ({
   oldSkills,
   portal,
```

Comment:
Is the portal prop really required? Seems we could replace it by simply setting `oldSkills` to an empty array.

Reply:
Hmm, but when a manager tries to reset the skill filters they previously set, we want to be able to know the previous skills they picked with the `oldSkills` prop. I added the portal prop to conditionally add that added filters section, in the manager's find skills modal, and also to remove the disabled "Already Added" button.

Reply:
Okay, so there will be other things later that only appear in the manager version? Makes sense.